config/credentials.yml.enc

```diff
@@ -1 +1 @@
-7qKS0Dyn981JowFwwTO7cvIXILfd65T8WbItWk+MDiVS7OyTtA2f0QiaNAdfWjvdnV17YW+kU1gkSON+FriPZZrtiqvkNBg/5ILdtC1BGzm+VkgDJeTHQ9oyop3GMKqvyGl7p/WuzNA6opZ+mnx6IcDR53jfe8tuvEQZQB2jvhfgYv2lOk8tOX1sOCe9N+oVuPneeBLKDyFltc+sS168c57N82ZjGq045PerULQ3xuGPTay9zBovqGFrJJZKDFUOlu5CFgXMcPwp6ZUZ6NH1rROQslQnx62QxuZNRQFkN3TiKpSqB65oWHGogIdugoSCSga4rUqDGPx2PnslldqtdDQ474voA6PyEvPhdEIHtWqds1GSEIrrldTQbh4zYu7gqHkwJrCbjljsJiQa5ArD1d82MbrpQrU0XBycE998YLQW+o5hODHMT29Y0fUaWjEduz4c21Be5yVmmBh4e/IVTqzYqygeTkNFCoD+MFPvilOZJpwgXT3Fh9B5y3SsrMyj1+E1wRUC+0JNa1Sz8o9hVYw8HDj8I/6P82XnnyALGy7qn30hXGiuhy4whmcnksVN/oyoPQWfSAN60gRoOIMTfzPFJlmT304ymQdhT1yLkmZQayNp0Nasm7d+QJo/EwvKGZwpkJYxDyqQfNrTLrqLKSj5TSqQ9vwBGPoThRq5McxWWskC4U9a4iYH76nd8B4UL3vUhTD1p8FUSDAPg1V2MDFKsGI9+4CKil17vgcayEKBx7VEDuL8pZ6J9inJ2JR9BSQJKjRaVvyFvSnG5BP4VwEIFx3zMSCvaEgryVpVB1jvJ7Llru6eW6d4nchKg9GSFqp12/r0BAx0U0VDC9aglqldyNmSWZJCFBcHCvGyrXcBBsNFJcvvkXSK1jNv2EwnXj+1UC8=--bD6LW0tN4VdbtCs4--kmV9URxpyZb3RuXkFtuX3Q==
\ No newline at end of file
+I0igOc2IzvRNXxvmYZFFcbZEkHIYMQbhPehKFy80/c+CkJyqGLcKZ9nq9JypOcso3lsbEHDKeyjMpXU297c86FTRx0e5BfAWrIHc/eYbg9YEsB3jJ5qaqxWmPTUq+SOKH2kGuW92KhaI96r9GJFmJelGWPvn+5fOmQMLMWaB86ZWO9povh97XkhTlZccksXYN2S/Q6Cv/pUIy4fwuajFl4dOYEkYqAa+dL6Q/SHOS8t2+/HhyGnSDuIT4fb0nFVpYQ4VPArRcXg6Y7E1BCfK3tInpJHwHtuyhtZJtr58u81+3Mx7dY3FYJu+onHOXIMqk5Ujdp+dp+ZTF5qlKAi7UDtD1W5/AyFsUZjKH2/PXfX8sEO0/CW1QllKMIfQz+9i30I3Z+Z7v1wpWRq28UzKoPLB9vZrB32VUx0JknfqLKj6mZD2u3kUpCcC7BxdY/yAptQpuKU8X2ZQAlIK25J9/PVosGGPU95RyyrviczmYMJ1ppweHCfAD7M7rgAYuu1rPIJv0gjGo+Rg7wLR7n0sWicR1/pdMP1pkVmIoR1WTSt1keTtdOX6Oe5eaZ6i9RlMwJoP9GBCFq3YLu1FEJLyplTPTqRRMCsmJIw5LpJqm7TYt10hXCKx2JDuOx6GUoJs78n9doyWnGf+IurEvnZBVvnostroShvmaV9mosbvkSMzbwQeQIXDlvysovNRBnuUwGmmCgRLvjTmpZR4b4Jn7T128dg3zqQVGc5+9lp+t/8merRwf1IniqfdYRhw17+uBusGMS/ikTXVSuSI0CnRRPcG2tvHUcF5+Db03R5NkTQ0f8dVXYuO21Uk08SbW0pG7MzGE3Bcjw81DTu0LPB6SKGHpY0+yaSEW1YowcrLKyPmsTzSEU1ve4lb5mHU1FvDdgdAXpeh6WiYu+X/7k9y2rlx/kUUZQCPY762oGrs+DOyO8fvpjdyJXrw4ci1h8H5VFhkrQLUlVWdM4CKszPtVuHxJA==--UOOAhmaEjtVaWXqe--4EVbkN7Puq1T8GPZ3Y135g==
```

Comment:
Bitte die `config/credentials.yml.template` Datei noch musterhaft Anpassen, damit auch diejenigen ohne einen passenden Master.key sehen können wie die Datei aussehen sollte

Reply:
Hinweis: bei unserem derzeitigen "Gitlab Deploy" wird auf dem Server vermutlich die `config/credentials.yml.enc` verwendet und nicht wie zu erwarten wäre die `config/credentials/production.yml.enc`.

Zur Sicherheit bitte daher Änderungen die online relevant sind in beide Dateien packen!

Reply:
Ja, dass habe ich mir schon gedacht. An den beiden Dateien habe ich aber nix verändert.

Reply:
Bitte denke noch an die Anpassung in `config/credentials.yml.template`